scripts/compose.py

```diff
@@ -2464,8 +2460,15 @@ def start_handler(self):
             args["version"] = self.SUPPORTED_VERSIONS.get(args["stack-version"], args["stack-version"])
 
         selections = set()
+        all_opbeans = args.get('run_all_opbeans')
+        any_opbeans = all_opbeans or any(v and k.startswith('enable_opbeans_') for k, v in args.items())
         for service in self.services:
-            if not args.get("disable_" + service.option_name()):
+            if args.get("enable_" + service.option_name()) \
```

Comment:
you could trow here some variable to make the whole thing more readable...

Reply:
excellent point. Done in 166fce39